src/marks/tree.js

```diff
@@ -27,10 +28,34 @@ export function tree(
     title = "node:path",
     dx,
     dy,
+    textAnchor,
+    textLayout,
     ...options
   } = {}
 ) {
   if (dx === undefined) dx = maybeTreeAnchor(options.treeAnchor).dx;
+  if (textAnchor !== undefined) throw new Error("textAnchor is not a configurable tree option");
+  textLayout = keyword(
+    textLayout === undefined ? (options.treeLayout === undefined ? "mirrored" : "normal") : textLayout,
```

Comment:
I think we should check whether treeLayout is Tree or Cluster here. I’ll make this change and then merge. (I was just thinking this would be nice for the D3 documentation!)